x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/AutoFollowIT.java

```diff
@@ -172,9 +180,54 @@ public void testAutoFollowParameterAreDelegated() throws Exception {
         });
     }
 
-    private void putAutoFollowPatterns(String... patterns) {
+    public void testConflictingPatterns() throws Exception {
```

Comment:
if two patterns match an index, why is that a problem? we used to be able to do it before no?

Reply:
So only one auto follow pattern would trigger the index following and when applying the second one would result in an error (same follow index already exists in follow cluster). Also which auto follow pattern gets applied first is undefined.

Reply:
I looked at it more and I realized I got confused - with the new model we can still specify multiple patterns in a single auto follow config (is this a better name?). All good. Thanks for explaining.

Reply:
> with the new model we can still specify multiple patterns in a single auto follow config (is this a better name?)

Yes, we can still do that.